pilot/pkg/networking/core/v1alpha3/gateway.go

```diff
@@ -190,7 +154,7 @@ func (configgen *ConfigGeneratorImpl) buildGatewayListeners(builder *ListenerBui
 		pluginParams := &plugin.InputParams{
 			Node:            builder.node,
 			Push:            builder.push,
-			ServiceInstance: si,
+			ServiceInstance: nil,
 		}
```

Comment:
is this valid? If so should we remove SseviceInsrances from plugin Paramus entirely?

Reply:
InputParams is not used by `OnOutboundListener`, but necessary for `inboundXXX`

If we want to remove this, i think we should define different `InputParams` for them

Reply:
I can update it as a separate one, if you feels that

Reply:
I see. Yeah let's not bundle it with this PR but it may be a good idea. We had bugs in the past where plugin expected certain field but not everywhere set it